pkg/load/load.go

```diff
@@ -186,22 +234,42 @@ func Config(path, unresolvedPath, registryPath string, info *ResolverInfo) (*api
 	return &configSpec, nil
 }
 
-func configFromResolver(info *ResolverInfo) (*api.ReleaseBuildConfiguration, error) {
+func configFromResolver(info *ResolverInfo, options configLoadOpt) (*api.ReleaseBuildConfiguration, error) {
```

Comment:
`ResolverInfo` is already a structure to hold the data we use to query the resolver - why not extend it?

Reply:
My intent is to explore the dynamic injection concept further in the future, I want to try things like "inject a workflow test". That leads to different resolver endpoints with different sets of parameters, and I don't think we want to have a single struct that's basially an union of all different parameters?

I'll try to invest a bit more effort into refactoring the client code, let's see if I can try to come up with something better.